community/knative/README.md

```diff
@@ -40,8 +40,8 @@ Disabling a chart will disable all charts below it in the chart structure. When
 enabled.
 - Istio - You should have Istio installed on your Kubernetes cluster. If you do not have it installed already you can do so by running the following commands:
 ```bash
-$ kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.1/istio-crds.yaml
-$ kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.1/istio.yaml
+$ kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.2/istio-crds.yaml
```

Comment:
nit: could we group these two cmds together, makes it shorter and consistent with community, like so..

```
$ kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.2/istio-crds.yaml && \
kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.2/istio.yaml
```